docs/core/tools/dotnet-new.md

```diff
@@ -499,10 +499,6 @@ Each project template may have additional options available. The core templates
 
   Includes BrowserLink in the project. Option not available in .NET Core 2.2 and 3.1 SDK.
 
-- **`-rrc|--razor-runtime-compilation`**
```

Comment:
is this no longer available?

Reply:
I tried running `dotnet new mvc --rrc`, and it said that isn't a thing. I don't work directly on the CLI, but I think that means it doesn't exist.

Reply:
`dotnew new webapp -rrc` works in 3.1 SDK

Reply:
![image](https://user-images.githubusercontent.com/12969783/80335624-b15d8200-8809-11ea-9bbd-7c9043c94be4.png)

Did I do something wrong?

![image](https://user-images.githubusercontent.com/12969783/80335638-c1756180-8809-11ea-911a-9bf50c52ed6e.png)

Reply:
@Forgind My mistake, I was testing on 3.1.102, it seems the removal happened between then and 3.1.200.

@scottaddie you  added the -rrc section to this doc recently in https://github.com/dotnet/docs/pull/17790 -- should we remove it now?

Reply:
@tdykstra The PR that introduced this template option is https://github.com/dotnet/aspnetcore/pull/18648. Specifically, the option is defined in the *template.json* file in that PR. The option is still included. I used the following steps to confirm that the option is available in the 3.1.101 and 3.1.200 SDKs:

1. `dotnet new global --sdk-version 3.1.101`
1. `dotnet new webapp -h`

Reply:
I know it's supported  iin 3.1.101, but why isn't it in 3.1.200?

Reply:
@tdykstra It is supported in 3.1.200. I tested this with both `dotnet new webapp` and `dotnet new mvc`.

![razor-compilation](https://user-images.githubusercontent.com/10702007/80440086-9b18fa00-88cd-11ea-8656-e4389335a316.gif)

You can see the option in the 3.1 release branch for [MVC](https://github.com/dotnet/aspnetcore/blob/release/3.1/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/.template.config/template.json#L297) and [Razor Pages](https://github.com/dotnet/aspnetcore/blob/release/3.1/src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json#L307).

Reply:
There must be some environmental requirement for it to show up then, because on my machine -- and apparently on @Forgind 's -- it's not there for 3.1.200

Reply:
@pranavkm Any idea why @tdykstra isn't seeing this template option when using the 3.1.200 SDK?

Reply:
We added support for this feature in 3.1.3. You'll need the 3.1.201 \ 3.1.103 SDKs to get support for this: https://dotnet.microsoft.com/download/dotnet-core/3.1. The SDK uses the most recent version of the templates installed. It could be likely that you have a newer SDK installed on your machine which is why you're seeing the 3.1.3 templates with an older SDK.

Reply:
Yes, I get it after installing 3.1.201.

@Forgind I suggest you restore the -rrc section and change "3.1" to "3.1.201" in "available since .NET Core 3.1 SDK"

Reply:
Sounds good.